normalised_power/normalised_power_function.R

```diff
@@ -1,15 +1,21 @@
-event_normalised_power <- function(combined_data, event_time, keep_site_id){
+event_normalised_power <- function(combined_data, event_time, keep_site_id) {
   event_time_data <- data.frame(filter(combined_data, Time > event_time - d & Time <= event_time))
   event_time_data <- setnames(event_time_data, c("site_performance_factor"), c("event_site_performance_factor"))
-  if (keep_site_id){
+  if (keep_site_id) {
     event_time_data <- select(event_time_data, site_id, event_site_performance_factor)
     event_time_data <- distinct(event_time_data)
-    combined_data <- left_join(combined_data, event_time_data, by=c("site_id"))
+    combined_data <- left_join(combined_data, event_time_data, by = c("site_id"))
   } else {
     event_time_data <- select(event_time_data, series, event_site_performance_factor)
-    combined_data <- left_join(combined_data, event_time_data, by=c("series"))
+    combined_data <- left_join(combined_data, event_time_data, by = c("series"))
   }
-  combined_data <- mutate(combined_data, Event_Normalised_Power_kW=ifelse(event_site_performance_factor>0.00001,
-             site_performance_factor/event_site_performance_factor, NA))
+  combined_data <- mutate(
```

Comment:
I think piping may be the best answer, as otherwise it'll lead to two different indentation styles. I'll go through and use piping whenever dplyr calls are used over multiple lines

Reply:
I've created a commit which fixes all of the other issues. Still going through this now.

Reply:
I've created a commit which changes most dplyr calls to use piping. The odd ones left without piping are generally one-liners, where it would be redundant to use piping.

Reply:
Thanks Christian!